Rom.py

```diff
@@ -37,15 +37,19 @@
 
 class LocalRom(object):
 
-    def __init__(self, file, patch=True, name=None, hash=None):
+    def __init__(self, file, patch=True, vanillaRom=None, name=None, hash=None):
         self.name = name
         self.hash = hash
         self.orig_buffer = None
+        self.vanillaBuffer = None
```

Comment:
orig_buffer is already the "original rom", aka vanilla
Let's not duplicate code and data.

Reply:
Then i have to load the vanilla rom into orig_buffer instead of vanillaBuffer and that's it